lib/Wheel.js

```diff
@@ -1,16 +1,34 @@
 class Wheel {
-    constructor(hasBeenSpun, currentValue, wheelData){
-        this.hasBeenSpun = false, 
-        this.currentValue = '', 
-        this.wheelData = []
+  constructor() {
```

Comment:
`Wheel` is missing a parameter for the current round based on how it is being invoked in line 15 of your `Round` file